docs/pages/includes/permission-warning.mdx

```diff
@@ -12,7 +12,8 @@
     numbered < `1024` (e.g. `443`).
   - Follow the "Principle of Least Privilege" (PoLP). Don't give users
     permissive roles when giving them more restrictive roles will do instead.
-    For example, assign users the built-in `access,editor` roles.
+    For example, don't assign users the built-in `access,editor` roles which give
+    them permissions to access and edit all cluster resources.
```

Comment:
```suggestion
    For example, don't assign users the built-in `access,editor` roles, which give
    them permissions to access and edit all cluster resources. Instead, define 
    RBAC roles with the minimum required permissions for each user and configure
    Access Requests for elevated permissions.
```

Suggested to give users an alternative rather than indicating what not to do